Tests/SwiftDocCTests/DocumentationService/DocumentationServer+DefaultTests.swift

```diff
@@ -87,7 +87,7 @@ class DocumentationServer_DefaultTests: XCTestCase {
         
         // Instead of using an `XCTestExpectation`, use a Boolean due to a bug in
         // swift-corelibs-xctest on Linux for expectations that get over-fulfilled
-        // https://bugs.swift.org/browse/SR-12575.
+        // https://github.com/apple/swift/issues/55020.
```

Comment:
Are there any other links to JIRA in the codebase, or references to SR numbers?

Reply:
This was the only bugs.swift link but I didn't look for SR numbers... Let me take a look for those.

Reply:
Updated here: https://github.com/apple/swift-docc/pull/233/commits/1ad108328cdc9df6f47845d637b5dbaa2bbe3180.